examples/counter-next/typegen.sh

```diff
@@ -9,11 +9,11 @@ rm -rf headers sources *.swift *.h *.modulemap
 popd
 popd
 
-cargo run --package shared --bin crux_cli --features cli -- \
-    codegen \
-        --java com.crux.example.counter.shared \
-        --swift SharedTypes \
-        --typescript shared_types
+# cargo run --package shared --bin crux_cli --features cli -- \
+#     codegen \
+#         --java com.crux.example.counter.shared \
+#         --swift SharedTypes \
+#         --typescript shared_types
```

Comment:
This can come back even with Facet at some point, right?

Reply:
Yes, indeed. In a separate PR we should update `crux_cli` to support facet typegen as well